### PR TITLE
add healthz, readyz probes to controller manager.

### LINKED ISSUE #602: Controller manager does not expose healthz/readyz probes
As a cluster admin running the `opentelemetry-operator`, I would like the controller manager to expose standard healthz & ready probes.

---

Currently the controller manager does not expose any probes and as such, can not be automatically managed by k8s in the event of application issues.

This is easily accomplished with the controller manager and something I intend to look into.

### PR DESCRIPTION
**Pull request by adriankostrubiak-tomtom**
add healthz, readyz probes to controller manager.

fixes #602

Signed-off-by: Adrian Kostrubiak adrian.kostrubiak@tomtom.com

-------

I'm unclear if there's a good way to add unit/etc tests for this change. If there is, I'd welcome any ideas or suggestions.

### CODE REVIEW

**Review by pavolloffay — APPROVED** [review]

File: main.go [hunk 1]
```diff
@@ -22,6 +22,8 @@ import (
 	"runtime"
 	"strings"
 
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
```
Comment (jpkrohling) [inline main.go:25]: The import order seems odd.

**Review by jpkrohling — COMMENTED** [review]
LGTM. The import order of a source file looks odd, once it's fixed, I'll merge this.

File: main.go (hunk 1 above)
Comment (adriankostrubiak-tomtom) [reply main.go]: good call, thanks. just pushed a change to resolve.

**Review by jpkrohling — APPROVED** [review]

**Comment by jpkrohling** [comment]
Auto-merging this. If it doesn't work, feel free to ping me and I'll merge it manually.

**Pull request merged**